testing/tests/DevExpress.viz.core.series/piePoint.tests.js

```diff
@@ -833,6 +834,26 @@ QUnit.test("Apply view with legend callback", function(assert) {
     assert.strictEqual(callback.callCount, 1);
 });
 
+QUnit.test("Update data when point was hidden", function(assert) {
+    this.options.visibilityChanged = noop;
+    let point = createPoint(this.series, { argument: "a1", value: 1 }, this.options);
+
+    point.hide();
+    point.updateData({ argument: "a1", value: 10 });
+
+    assert.strictEqual(point.isVisible(), false);
+});
+
+QUnit.test("Add new argument to dataSource when some points were hidden", function(assert) {
```

Comment:
Test caption should be something like "Point should reset hidden state if argument changed"